q2mm/datatypes.py

```diff
@@ -415,7 +422,7 @@ def import_ff(self, path=None, sub_search='OPT'):
                             comment = line[COM_POS_START:].strip()
                             self.sub_names.append(comment)
                         parm_cols = line[P_1_START:P_3_END]
-                        parm_cols = map(float, parm_cols.split())
+                        parm_cols = list(map(float, parm_cols.split()))
```

Comment:
Assuming we're using Python 3, I wonder if this (convert to list and extend) or iterating and appending would be faster.

Reply:
The later would likely be more efficient with memory.

Reply:
I'm not sure how much of an issue it would be in terms of memory or computation efficiency. If it's not slow and doesn't take too much memory, then it's probably not worth worrying about too much.